pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/sql.tsx

```diff
@@ -138,7 +138,7 @@ export default function (props: GraphDashboardProps) {
     </LineGraph>,
 
     <LineGraph
-      title="Service Latency: DistSQL, 99th percentile"
+      title="Service Latency: Distributed SQL, 99th percentile"
```

Comment:
I'd be in favor of just dropping these charts. @piyush-singh @Amruta-Ranade thoughts?

Reply:
I agree. Given that you have the statements page if you want more of a breakdown, I don't think users really need this. The metric is still being recorded; it's accessible via the custom graph page.

Reply:
Agreed, especially since we have made a concerted effort to remove language that mentions DistSQL elsewhere. Good to remove on my end!

Reply:
Cool, thanks. They're gone!